deps-mvn-indexing/src/main/clojure/com/connexta/osgeyes/index/core.clj

```diff
@@ -67,4 +68,76 @@
   (count (gather-hierarchy "ddf" "2.19.5"))
   (map #(artifact-info->map % #{:attrs}) (do-gather-hierarchy "ddf" "ddf" "2.19.5"))
   (map #(.toString %) (do-gather-hierarchy "ddf" "ddf" "2.19.5"))
-  (close-indexer!))
\ No newline at end of file
+  (close-indexer!))
+
+(def test-data
```

Comment:
Check the other Clojure source files to see the comment pattern I leave behind to deliminate groups of functions within the same namespace, and add an instance of that grouping header here to denote package search related functions.